webcompat/static/js/lib/flashed-messages.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This file is only included in the page when the Flask view is rendered with
+// a flash message. It's responsible for triggering that message as defined
+// in flash-message.js. See layout.html for how category and message data
+// attributes are set.
+
+var currentScript = document.currentScript;
```

Comment:
@wisniewskit suggested that if we stick the category and message somewhere else like body or html (or main), we don't need to use `currentScript` and it will work in IE. 👍